requirements/constraints.txt

```diff
@@ -0,0 +1,17 @@
+# Version constraints for pip-installation.
+#
+# This file doesn't install any packages. It specifies version constraints
+# that will be applied if a package is needed.
+#
+# When pinning something here, please provide an explanation of why.  Ideally,
+# link to other information that will help people in the future to remove the
+# pin when possible.  Writing an issue against the offending project and
+# linking to it here is good.
+
+# TODO: Many pinned dependencies should be unpinned and/or moved to this constraints file.
+
+# pip 5.0.0 only works with pip 20+ and all our VMs and containers are still installing 19.3.1 by default
+pip-tools<5.0.0
+
+# Fix tox
```

Comment:
Could add a more descriptive reason to add this constraint? if you are not sure about that you should mention that it was a constraint that was added before the implementation of oep-18

Reply:
@morenol I am not even sure if this constraint is actually required. Look this commit https://github.com/edx/openedx-calc/commit/9b662aca5f41c5aeff3b1c81f019f2d33ffa72ee
After that some constraints were removed https://github.com/edx/openedx-calc/commit/ea6f5e4d749277279078a7a8ba490a91dbddd200. May I just remove this constraint ?

Reply:
Let's try that